vm/message/Cargo.toml

```diff
@@ -22,6 +22,7 @@ serde = { version = "1.0", features = ["derive"] }
 base64 = { version = "0.12.1", optional = true }
 forest_json_utils = { path = "../../utils/json_utils", optional = true, version = "0.1" }
 num-traits = "0.2"
+types = { package = "fil_types", path = "../../types"}
```

Comment:
Nit: `types = { package = "fil_types", path = "../../types" }` add space in the back 🙃

Reply:
i hate you. need to run ci again now 😂